pom.xml

```diff
@@ -232,6 +233,11 @@
           <artifactId>maven-failsafe-plugin</artifactId>
           <version>${version.org.apache.maven.failsafe}</version>
         </plugin>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>keytool-maven-plugin</artifactId>
```

Comment:
What is this used for?

Reply:
In tests is this plugin used to import certificate to use HTTPS routes.
PR with tests will be open after merging changes to framework.